server/opts.go

```diff
@@ -804,6 +845,11 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 		o.ConnectErrorReports = int(v.(int64))
 	case "reconnect_error_reports":
 		o.ReconnectErrorReports = int(v.(int64))
+	case "websocket":
```

Comment:
think an alias here like `"websocket", "ws"` would be great

Reply:
Done